README.md

```diff
@@ -28,31 +28,32 @@ selection of targets to use for communication with SRV-located services.
 It presents this service in the following interface:
 
 ```rust
-use srv_rs::{client::SrvClient, resolver::libresolv::LibResolv};
+use srv_rs::{client::{SrvClient, Execution}, resolver::libresolv::LibResolv};
 let client = SrvClient::<LibResolv>::new("_http._tcp.example.com");
-srv_rs::execute!(client, |address: &http::Uri| async move {
+client.execute_one(Execution::Serial, |address: http::Uri| async move {
```

Comment:
So, standard disclaimer about being bad at naming...

My instinct is that you give the most commonly used API the most generic name, then distinguish the variations for less common use cases with an adjective/suffix/whatever. My guess is that consumers would overwhelmingly want what is currently called `execute_one` (usually with Serial execution, but sometimes Concurrent) and that the streaming method that hits all targets is much rarer/more specialized. I might call that one `execute_stream` or `execute_for_all` or something and give `execute` to what is currently `execute_one`?

That or ignore me, but you asked for naming input.

Reply:
Agreed. I was hesitant to use `execute_stream` because I thought it might not be intuitive, but neither is `execute_one` so I might as well prioritize the more common case.